### PR TITLE
Initial Kotlin support + add 'deploy' in the 'help' list of commands

### PR DESCRIPTION
**Pull request by delabassee**
'deploy' is simply missing from the help

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ LEARN MORE:
 		lambda(),
 		version(),
 		calls(),
+		deploy(),
```
Comment (treeder) [inline main.go]: This is already there: https://github.com/fnproject/cli/pull/120/files#diff-7ddfb3e035b42cd70649cc33393fe32cR77

Comment (treeder) [reply main.go]: Oh, I see, it's not showing --help.

**Comment by treeder** [comment]
Should probably mention that this PR also adds Kotlin support. ;)

**Comment by delabassee** [comment]
PR title fixed!
That's what happens when you submit a PR during a conf.call :expressionless:

**Pull request merged**